packages/common/src/grpc-service.js

```diff
@@ -161,6 +161,11 @@ function GrpcService(config, options) {
 
   this.grpcMetadata = new grpc.Metadata();
 
+  this.grpcMetadata.add('User-Agent', [
+    config.packageJson.name.replace('@google-cloud', 'gcloud-node'),
```

Comment:
In case I wasn't being clear, I originally thought the service name was important to include with the version. Mapping `gcloud-node/version` to `google-cloud/bigtable@version` might not be obvious, since gcloud-node/google-cloud-node might share that same version.

To use the UA string correctly, I think we're supposed to describe a "product" with its version. The product is the module, so describing it as `gcloud-node` isn't accurate. But this stuff is being talked about in https://github.com/GoogleCloudPlatform/gcloud-common/issues/179, so I'm just waiting to see which way they want to go.